### PR TITLE
doc: Add and update auth subsystem command description and clean up some nonexistent command descriptions.

### PR DESCRIPTION
**Pull request by scienceluo**
doc: Add and update auth subsystem command description and clean up some nonexistent command descriptions.

Signed-off-by: Luo Kexue <luo.kexue@zte.com.cn>

### CODE REVIEW

**Comment by scienceluo** [comment]
Jenkins retest this please

**Comment by scienceluo** [comment]
retest this please

**Comment by scienceluo** [comment]
@joscollin Can you help me to review this? Thank you.

**Comment by scienceluo** [comment]
@tchaikov Can you help me to review this? Thank you.

File: doc/rados/operations/control.rst [hunk 1]
```diff
@@ -44,14 +44,53 @@ or not it is in the quorum. ::
...
+
+To get or add key for ``entity`` from system/caps pairs specified, execute the following::
+
+        ceph auth get-or-create-key <entity> {<caps> [<caps>...]}
+
+To import auth info from keyring file, execute the following::
+
+        ceph auth import {--in-file|-i} {path-to-keyring}
+
+To display key info fo ``entity``, execute the following::
```
Comment (scienceluo) [inline doc/rados/operations/control.rst]: Yep, thanks.

**Comment by scienceluo** [comment]
retest this please.

**Comment by joscollin** [comment]
Jenkins retest this please

**Comment by scienceluo** [comment]
retest this please

File: doc/rados/operations/control.rst [hunk 2]
```diff
@@ -44,14 +44,53 @@ or not it is in the quorum. ::
...
+
+        ceph auth import {--in-file|-i} {path-to-keyring}
+
+Display key info for ``entity``. ::
+
+        ceph auth print-key <entity>
+
+Remove all caps for ``entity``. ::
+
+        ceph auth rm <entity>
```
Comment (joscollin) [inline doc/rados/operations/control.rst]: Is this a duplicate of `ceph auth del <entity>` ? According to me `del` and `rm` should be standardized / unified, but that should be a different PR.

File: doc/rados/operations/control.rst [hunk 3]
```diff
@@ -44,14 +44,53 @@ or not it is in the quorum. ::
 Authentication Subsystem
 ========================
 
-To add a keyring for an OSD, execute the following::
+Add auth info for ``entity`` from input file, or random key if no input is given, and/or any caps specified. ::
```
Comment (tchaikov) [inline doc/rados/operations/control.rst]: i don't think duplicating the section of https://github.com/ceph/ceph/blob/master/doc/man/8/ceph.rst#auth would help. instead you can just add a hyper link point to it.

File: doc/rados/operations/control.rst [hunk 4]
```diff
@@ -141,16 +180,12 @@ Remove an existing bucket from the CRUSH map. ::
 
 Move an existing bucket from one position in the hierarchy to another.  ::
 
-   ceph osd crush move {id} {loc1} [{loc2} ...]
+        ceph osd crush move {id} {loc1} [{loc2} ...]
```
Comment (tchaikov) [inline doc/rados/operations/control.rst]: why?

File: doc/rados/operations/control.rst (hunk 3 above)
Comment (scienceluo) [inline doc/rados/operations/control.rst]: @tchaikov Ok, I see. I will close this pr. Thanks for your time any way.

File: doc/rados/operations/control.rst (hunk 4 above)
Comment (scienceluo) [inline doc/rados/operations/control.rst]: @tchaikov Align with the previous command.

File: doc/rados/operations/control.rst (hunk 2 above)
Comment (scienceluo) [inline doc/rados/operations/control.rst]: Maybe yes. And I will adopt tchaikov's advice to close this pr. Thank you all the same for your kind review. @joscollin

Comment (joscollin) [inline doc/rados/operations/control.rst]: @scienceluo No problem.
